File: Tarea05/ejercicio6.c
#include <stdio.h>

int main(){
	int conv;
	int horas, minutos;
	int final;
	printf("Ingresa 1 para convertir horas y minutos a segundos y 2 para convertir segundos a horas y minutos\n");
	scanf("%i", &conv);
	if (conv == 1){
		printf("Ingresa las horas\n");
		scanf("%i", &horas);
		printf("Ingresa los minutos\n");
		scanf("%i", &minutos);
		final = minutos*60 + horas*3600;
		printf("%i horas con %i minutos son %i segundos\n", horas, minutos, final);
	}
	else{
		int seg, segundos;
		printf("Ingresa los segundos\n");
		scanf("%i", &seg);
		horas = seg/3600;
		segundos = seg % 60;
		minutos = (seg-horas*3600)/60;
		printf("%i segundos son %i horas con %i minutos con %i segundos\n", seg, horas, minutos, segundos);
	}
	return 0;
}

File: Tarea05/ejercicio6.py
conv = float(input("Ingresa 1 para convertir horas y minutos a segundos y 2 para convertir segundos a horas y minutos\n"))
if conv == 1:
   horas = int(input("Ingresa las horas\n"))
   minutos = int(input("Ingresa los minutos\n"))
   final = minutos*60 + horas*3600
   print("%f horas con %f minutos son %d segundos"%(horas, minutos, final))
else:
   seg = int(input("Ingresa los segundos\n"))
   horas = int(seg/3600)
   segundos = seg % 60
   minutos = int((seg-horas*3600)/60)
   print("%d segundos son %d horas con %d minutos con %d segundos"%(seg, horas, minutos, segundos))
